lib/elixir/lib/calendar.ex

```diff
@@ -349,10 +439,18 @@ defmodule Date do
     {year, month, day}
   end
 
+
```

Comment:
Please remove the previous clause. We don't need it since `convert!(date_in_iso, Calendar.ISO)` should be a noop.